lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -210,28 +205,30 @@ class CategoryRenderer {
    * Renders the group container for a group of audits. Individual audit elements can be added
    * directly to the returned element.
    * @param {!ReportRenderer.GroupJSON} group
+   * @param {{expandable: boolean}} opts
    * @return {!HTMLDetailsElement}
```

Comment:
should drop `HTMLDetailsElement` for `Element` (which breaks `el.open` but old-type-inference closure doesn't  catch that)